numpy/random/tests/test_smoke.py

```diff
@@ -793,6 +793,10 @@ def test_default_is_pcg64(self):
         # a deprecation cycle to move to a different function.
         assert_(isinstance(self.rg.bit_generator, PCG64))
 
+    def test_expm1(self):
+        np.random.default_rng(12345)
+        assert_(np.random.default_rng(1e99) > 0.0)) 
+
```

Comment:
There is a stray ')' at the end of the line

Reply:
Thanks. Removed.

Reply:
The test should be something else, this creates a new `Generator`

Reply:
```
rng = np.random.default_rng(12345)
assert_(rng.pareto(1e99) > 0.0)
```